pagopa/pn-frontend
# Feat/pn 385 visualizzazione notifiche come delegato

## Short description
This PR adds routing, component changes and translations to show the notification dashboard and detail received as a delegate for a company(persona giuridica)

## List of changes proposed in this pull request
- Adds behaviour change for the sidebar when a user is delegated by at least one person or not
- Adds routing for the notifications dashboard and detail page
- Adds changes to handle both their own and the delegated notifications to Notifiche page, DesktopNotifications, mobileNotifications and Notifications api
- Adds translations for delegated notifcations page
- Fixes tests

## How to test
At the moment doesn't seem possible to delegate a user without groups or show notifications that have a group assigned.
However if a pg user has been delegated from someone, in the sidebar it should be possible to see Notifiche delegate in the sidebar, by clicking on it the app should navigate to the page that shows an aggregate of all the notifications received by our delegators.
Clicking on a notification it should navigate to the notification detail for a delegator's notification with a url in the form of:
`notifiche-delegato/:mandateId/:iun/dettaglio`
